problems/prob1.rb

```diff
@@ -0,0 +1,10 @@
+#Given an integer array of size N, compute
+#the sum of all even numbers in this array
+
+def sum_of_even_numbers(numbers_array)
+  if numbers_array.nil? || numbers_array.empty? || numbers_array.any? { |e| e == nil  }
+    return nil
+  else
+    return numbers_array.select{|x| x % 2 == 0}.sum
```

Comment:
This is almost perfect, but ruby makes this __even__ nicer, check the the integere [documentation](https://ruby-doc.org/core-2.6.2/Integer.html) and see if you notice something usefull there.

Reply:
```suggestion
    return numbers_array.select{|x| x.even?}.sum
```